### PR TITLE
gow: add page

### PR DESCRIPTION
**Pull request by michaelcale**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Review by Waples — APPROVED** [review]

File: pages/common/gow.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Go Watch
```
Comment (navarroaxel) [inline pages/common/gow.md:1]: The page title should be the command name
```suggestion
# gow
```

File: pages/common/gow.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+> Watches Go files and restarts the app on changes.
+> More information: <https://github.com/mitranim/gow>.
+
+- Start and watch current directory:
```
Comment (navarroaxel) [inline pages/common/gow.md:6]:
```suggestion
- Start and watch the current directory:
```

File: pages/common/gow.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`gow run .`
+
+- Pass arguments to the app
+
+`gow run . {{arg1 arg2 ...}}`
+
+- Watch subdirectories in verbose mode
+
+`gow -v -w={{dir1,dir2 ...}} run .`
```
Comment (navarroaxel) [inline pages/common/gow.md:16]:
```suggestion
`gow -v -w={{path/to/directory1,path/to/directory2,...}} run .`
```

File: pages/common/gow.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+> Watches Go files and restarts the app on changes.
+> More information: <https://github.com/mitranim/gow>.
+
+- Start and watch current directory:
+
+`gow run .`
+
+- Pass arguments to the app
+
+`gow run . {{arg1 arg2 ...}}`
```
Comment (navarroaxel) [inline pages/common/gow.md:12]: We don't abbrev 
```suggestion
`gow run . {{argument1 argument2 ...}}`
```

File: pages/common/gow.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Watch subdirectories in verbose mode
+
+`gow -v -w={{dir1,dir2 ...}} run .`
+
+- Specify file extensions to watch
```
Comment (navarroaxel) [inline pages/common/gow.md:18]:
```suggestion
- Watch the specified file extensions:
```

File: pages/common/gow.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+`gow -e={{go,html ...}} run .`
```
Comment (navarroaxel) [inline pages/common/gow.md:20]:
```suggestion
`gow -e={{go,html}} run .`
```

File: pages/common/gow.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+- Help
```
Comment (navarroaxel) [inline pages/common/gow.md:22]:
```suggestion
- Display help:
```

File: pages/common/gow.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
+# Go Watch
+
+> Watches Go files and restarts the app on changes.
+> More information: <https://github.com/mitranim/gow>.
+
+- Start and watch current directory:
+
+`gow run .`
+
+- Pass arguments to the app
```
Comment (navarroaxel) [inline pages/common/gow.md:10]:
```suggestion
- Start the application with the specified arguments:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**